### ISSUE TITLE
add confirmation prompt when playing units onto a full battlefield or when no more conjurations exist

### ISSUE THREAD

**Issue by bdwain** [Ashteki/ashteki#509]
It's really easy to accidentally play a unit onto a full battlefield, which always needs to be undone via manual mode. It's also easy to accidentally play a conjuration when you have no more. 

It would be good to add a confirmation prompt when doing either of the above things (via ally or conjuration) since you usually would not want to do that (though you would sometimes).

It might make sense to also code certain cards to not show the prompt in certain circumstances, such as steadfast guardian, though it's probably fine to skip that feature since it's so limited.

**Comment by danj3000** [comment]
I agree mostly, but think it should wait until after #496, which may prevent a number of users attempting it incorrectly.

Allies shouldn't be playable/selectable if the BF is full, but the conjuration/ summon angle is more nuanced and probably wants to be done with settings to switch it off for experienced players

I'll create a new issue to do the Allies part separately

**Comment by danj3000** [comment]
Handling this via visual cues rather than introducing confirmation
